uac-client2/src/main/java/com/bazaarvoice/emodb/uac/client2/UserAccessControlClientFactory.java

```diff
@@ -27,6 +53,109 @@ private UserAccessControlClientFactory(URI endPoint, Client jerseyClient) {
     }
 
     public AuthUserAccessControl create() {
-        return new UserAccessControlClient(_uri, _client);
+        return new UserAccessControlClient(_uri, _client, createRetryPolicy());
+    }
+
+    public RetryPolicy<Object> createRetryPolicy() {
```

Comment:
this method is duplicated in all jersey2 clients.
any objections to moving to the emodb-common-jersey2-client maven module?